src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const lines = sourceString.split(';');
+  const cssStyles = {};
+
+  for (let i = 0; i < lines.length; i++) {
+    const line = lines[i].trim();
+
+    if (!line) {
+      continue;
+    }
+
+    const lineSplitted = line.split(':');
+    const [cssProperty, cssValue] = lineSplitted.map((el) => el.trim());
+
+    if (cssProperty && cssValue) {
+      cssStyles[cssProperty] = cssValue;
+    }
+  }
+
+  return cssStyles;
```

Comment:
The function `convertToObject` uses a for loop to iterate over the style lines. According to the checklist, you should avoid using loops and instead use iteration methods such as `map`, `filter`, or `reduce`. Consider refactoring the code to use these methods for a more functional approach, which can lead to more concise and readable code.